crates/symbolicator/src/services/symbolication.rs

```diff
@@ -571,17 +571,13 @@ fn object_info_from_minidump_module_rust_minidump(
     ty: ObjectType,
     module: &MinidumpModule,
 ) -> RawObjectInfo {
-    let code_id = module.code_identifier().to_string();
+    let code_id = module.code_identifier();
 
     RawObjectInfo {
         ty,
-        // TODO: shouldn't this be None if code_id is empty?
```

Comment:
this is for my own personal curiosity, but what was the answer to this question?

Reply:
code_id can't be empty because of the new API which guarantees to give you one.  But, hum, good question.  Maybe this isn't true because you can still create those to be empty.  I'll do this as a followup up though because this could be messy.